js_modules/dagster-ui/packages/ui-core/src/runs/TickTagForRun.tsx

```diff
@@ -19,14 +19,14 @@ export const TickTagForRun = ({instigationSelector, instigationType, tickId}: Pr
   return (
     <>
       <Tag icon={icon}>
-        <span>
-          Launched by{' '}
+        <Box flex={{direction: 'row'}}>
```

Comment:
`alignItems: 'center'` ?

Reply:
Looks like it's okay as `normal`, so I'll just keep it this way for now and we can keep an eye on it.